src/tests/test_accesses.py

```diff
@@ -62,100 +58,3 @@ def test_fetch_accesses(test_app, monkeypatch):
     assert response.status_code == 200
     assert response.json() == [{k: v for k, v in entry.items() if k != "hashed_password"}
                                for entry in mock_access_table]
```

Comment:
If we remove the unittests for the routes, we need to adapt them to test the crud.accesses functions. I don't think it would be wise to decrease coverage by that much